litellm/llms/bedrock.py

```diff
@@ -605,6 +605,9 @@ def init_bedrock_client(
         aws_web_identity_token,
     ) = params_to_check
 
+    # SSL certificates (a.k.a CA bundle) used to verify the identity of requested hosts.
+    ssl_verify = os.getenv("SSL_VERIFY", litellm.ssl_verify)
```

Comment:
bedrock.py is a deprecated file. all bedrock calls are routed via bedrock_httpx.py now.

Reply:
I remember you saying this, but it seemed that embedding calls were handled through this deprecated client. I'll check it again to make sure I didn't miss anything. Thanks for pointing it out.